pkg/shell/credentials.jsx

```diff
@@ -274,7 +274,7 @@
                         aria-label={_("Password tip")}
                         bodyContent={_("Tip: Make your key password match your login password to automatically authenticate against other systems.")}
                     >
-                        <button className="pf-c-form__group-label-help ct-icon-info-circle"
+                        <button className="pf-v5-c-form__group-label-help ct-icon-info-circle"
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230516-073248-d32f52be-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)